src/attributes.jl

```diff
@@ -183,7 +189,7 @@ function get end
 # We want to avoid being too specific in the type arguments to avoid method ambiguity.
 # For model, get(::ModelLike, ::AbstractVariableAttribute, ::Vector{VariableIndex}) would not allow
 # to define get(::SomeModel, ::AnyAttribute, ::Vector)
-get(model::ModelLike, attr::AnyAttribute, idxs::Vector) = get.(model, Ref(attr), idxs)
+get(model::ModelLike, attr::AnyAttribute, idxs::Vector) = get.(model, attr, idxs)
```

Comment:
Do we still need this fallback or should users just broadcast themselves?

Reply:
Some solvers may do something faster,  e.g. if they have a fixed cost that can be shared when getting by group